src/meta/buildTags.tsx

```diff
@@ -605,12 +605,12 @@ const buildTags = (config: BuildTagsParams) => {
       );
     }
 
-    if (config.openGraph.site_name) {
+    if (config.openGraph.siteName) {
```

Comment:
```suggestion
    if (config.openGraph.siteName || config.openGraph.site_name) {
```

This will allow this change to be backwards compatible.